contents/python/numpy.md

```diff
@@ -0,0 +1,167 @@
+<frontmatter>
+  title: NumPy
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
+# NumPy
+
+Author: [Jeremy Tan](https://github.com/Parcly-Taxel)
+
+<box id="article-toc">
+* [What NumPy can and cannot do for you](#what-numpy-can-and-cannot-do-for-you)
+* [The array type](#the-array-type)
+* [Library functions‎](#library-functions)
+* [Getting started‎](#getting-started)
+* [Resources‎](#resources)
+</box>
```

Comment:
It's nicer if the headings show a high-level narrative, and are consistently phrased. e.g.,
* What is NumPy
* How it works
  * The array type‎
  * Library functions‎
* Getting started‎
* Resources‎